src/_zkapauthorizer/tests/__init__.py

```diff
@@ -51,7 +51,7 @@ def _configure_hypothesis():
 
     settings.register_profile(
         "fast",
-        max_examples=2,
+        max_examples=3,
```

Comment:
intentional?

Reply:
oh, oops .. was trying to see how much difference changing that makes (answer: unknowable, at these sizes).